backup/__init__.py

```diff
@@ -0,0 +1,52 @@
+import dbbackup
+import datetime
+import os
+import sys
+import time
+
+from django.core.management import call_command
+
+from fle_utils.chronograph.models import Job
+
+backup_dirpath = expanduser("~")
+backup_dirpath = os.path.join(backup_dirpath, 'ka-lite-backups')
+
+def file_rename():
+    filename ='default.backup'
+    time = datetime.datetime.now().ctime()
+    path = os.path.join(backup_dirpath, filename)
+    target = os.path.join(backup_dirpath, time + '.backup')
+
+    if path.exists():
+        os.rename(path, target)
+
+def setup_backup():
+    try:
+        command = call_command('dbbackup')
+        force_job(command, frequency="DAILY")
+    except KeyError
+
+def restore_list():
+    file_list = os.listdir(backup_dirpath) #Retrieve the filenames present in the backup_dirpath
+    filelist = []
+    filenumber = 0
+
+    for eachfile in file_list:
+        filelist.append(eachfile) #Store files one by one
+        print filelist.index(eachfile), eachfile #show file number: index
+        filenumber += 1
```

Comment:
Also, I prefer that we remove all input and output out of this `__init__.py` file and into the `restorebackup` management command. Instead `restore_list` should just take in the name of the backup and then restore that backup. The logic for printing out the list of backups and then asking for user input should be confined to the `restorebackup` management command.